src/components/WixUiTpaConfig/WixUiTpaConfig.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
```

Comment:
This is how I see the wanted api:

```jsx
import {Provider} from 'wix-ui-tpa/Provider';
import {Card} from 'wix-ui-tpa/Card';

const App = ({WixSdk}) => (
  <Provider mobile={WixSdk.isMobile()}>
     <Card/>
  </Provider>
)
```

The `mobile` prop should propagate internally to the `Card` component and shouldn't be a public API.